modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/soy/BaseClayCard.java

```diff
@@ -36,6 +36,10 @@ public default String getComponentId() {
 		return null;
 	}
 
+	public default String getCssClass() {
```

Comment:
@markocikos this is how I resolved this (step 1)

Reply:
@carloslancha I guess it does makes sense to extend object. Maybe deprecate `getElementClasses` along with it?

Reply:
`BaseClayCard` is an interface class with implementations. I am a bit surprised Java compiler allows this :thinking:

Reply:
> Edit: TIL about default methods.

Share it with others!

I think it's available since Java 8 ;)